SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/TetrahedronSetTopologyContainer.cpp

```diff
@@ -1207,4 +1207,59 @@ void TetrahedronSetTopologyContainer::updateTopologyEngineGraph()
     TriangleSetTopologyContainer::updateTopologyEngineGraph();
 }
 
+std::ostream& operator<< (std::ostream& out, const TetrahedronSetTopologyContainer& t)
```

Comment:
I'm not sure this make really sens to have iostream operator for a TopologyContainer.
I would be curious to see where it is used.

Reply:
moreover it is not using all the "modifier" methods (AddTetrahedron, etc.) that handle change in the topology and propagate the events

Reply:
I didn't  wanted to change the existing code... just move the stream out of the .h. Refactoring this part of sofa code would be very nice but it is not my topic right now and I will not pick a new one until I finished the one I'm in ;)